### PR TITLE
Feature/advanced plotting/measurement table [2/6]

### PR DESCRIPTION
**Pull request by DJAntivenom**
# Measurement Table

This PR updates the measurement table to better handle sweeps and to allow individual callbacks to be notified when the selection changes.

This will then be used by the new plotting views to update themselves.

### CODE REVIEW

File: LabExT/View/MeasurementTable.py [hunk 1]
```diff
@@ -114,9 +168,45 @@ def __setup__(self):
...
+                measurements = self._hashes_of_meas[item_id_or_ids] if item_id_or_ids in self._hashes_of_meas else ""
+            else:
+                measurements = [self._hashes_of_meas[item] for item in item_id_or_ids if item in self._hashes_of_meas]
+            callback(
+                (
+                    item_id_or_ids,
+                    is_checked,
+                    [(hash, any([(hash in child) for child in checked_items])) for hash in self._hashes_of_meas.keys()],
+                    measurements,
+                )
```
Comment (marcoep) [inline LabExT/View/MeasurementTable.py:204]: Could you please explain the function header you are using for the callback functions and its arguments?

Comment (marcoep) [reply LabExT/View/MeasurementTable.py:204]: Yes, I think a comment would make sense here. thanks

**Review by marcoep — APPROVED** [review]

File: LabExT/View/MeasurementTable.py (hunk 1 above)
Comment (DJAntivenom) [reply LabExT/View/MeasurementTable.py:214]: Done in 1ee47614120acd6a5498d43e9cedcaeee65b0d7a.

**Comment by DJAntivenom** [comment]
This PR is superseded by #208

**Pull request closed without merging**